src/LabelSelector/LabelChip/LabelChip.tsx

```diff
@@ -12,31 +13,37 @@ export default function LabelChip({
   size = "medium",
   variant = "filled",
   visible = true,
+  selected = false,
+  description = "",
+  showIcon = false,
   ...props
 }: LabelChipProps) {
   // return the styled chip component
   return (
-    <Chip
-      {...props}
-      className="label-chip"
-      clickable={clickable}
-      sx={{
-        "&:hover": {
-          backgroundColor: clickable ? darken(color, 0.2) : color
-        },
-        backgroundColor: color,
-        color: theme => theme.palette.getContrastText(color),
-        [`& .${chipClasses.deleteIcon}`]: {
-          color: theme => alpha(theme.palette.getContrastText(color), 0.7)
-        },
-        [`& .${chipClasses.deleteIcon}:hover`]: {
-          color: theme => theme.palette.getContrastText(color)
-        },
-        visibility: visible ? "visible" : "hidden"
-      }}
-      label={<NoWrapTypography variant="inherit">{label}</NoWrapTypography>}
-      size={size}
-      variant={variant}
-    />
+    <Tooltip title={description} placement="bottom-start">
+      <Chip
+        {...props}
+        className="label-chip"
+        clickable={clickable}
+        icon={showIcon && selected ? <DoneIcon color="inherit" /> : undefined}
```

Comment:
Whats the use case for selected without icon? This scenario wasn't accounted for in the original ticket so I'm wondering why we've added a third property.

Reply:
Like now I can set `showIcon` to true, but no icon appears until I also set `selected` to true. Suggest we stick with the original `selected` proposal unless there is new information.